src/h5web/visualizations/heatmap/HeatmapVis.module.css

```diff
@@ -32,6 +32,29 @@
   display: flex;
 }
 
+.rightArea {
+  grid-area: right-area;
+}
+
+.colorBar {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 95%;
+}
+
+.colorBarAxis {
+  display: block;
+  overflow: visible;
+  width: min-content;
```

Comment:
As per one of my previous comments, this gives the axis a width of 0, which is why the colour bar is centred horizontally as if the axis wasn't there. Just put a fixed width on the `svg` element in the JSX to fix the centring.